Realm/RLMUtil.mm

```diff
@@ -421,9 +417,19 @@ BOOL RLMIsRunningInPlayground() {
         case RLMPropertyTypeObject: {
             // If we are unboxing an object and it is unmanaged, we need to
             // add it to the Realm.
-            if (RLMObjectBase *objBase = RLMDynamicCast<RLMObjectBase>(v); !objBase->_realm) {
-                RLMAddObjectToRealm(objBase, realm, RLMUpdatePolicyError);
+            if (RLMObjectBase *objBase = RLMDynamicCast<RLMObjectBase>(v); !objBase->_realm
+                && createPolicy.create) {
```

Comment:
Indenting like this makes the second line of the conditional loop like part of the if body.